src/helpers/index.js

```diff
@@ -27,5 +30,7 @@ export {
   fromHex,
   radspec,
   transformTime,
-  tokenAmount
+  tokenAmount,
+  blockTime,
+  isBlockMined
```

Comment:
Not sure why the `defaultHelpers` list is unalphabetized, but let's alphabetize both of these lists :)